pkg/scheduler/cache/cache.go

```diff
@@ -50,7 +50,7 @@ import (
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog"
 	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
-	volumescheduling "k8s.io/kubernetes/pkg/scheduler/framework/plugins/volumebinding"
+	volumescheduling "volcano.sh/volcano/pkg/scheduler/plugins/volumebinding"
```

Comment:
cache should not depend plugin

Reply:
suggest add capabilities directory like pkg/scheduler/capabilities/volumebinding and put our volumebinding there